cheroot/test/test_ssl.py

```diff
@@ -729,3 +749,69 @@ def test_http_over_https_error(
         format(**locals())
     )
     assert expected_error_text in err_text
+
+
+@pytest.mark.parametrize(  # noqa: C901  # FIXME
+    'adapter_type',
+    (
+        'builtin',
+        'pyopenssl',
+    ),
+)
+@pytest.mark.parametrize(
+    'ip_addr',
+    (
+        ANY_INTERFACE_IPV4,
+        pytest.param(ANY_INTERFACE_IPV6, marks=missing_ipv6),
+    ),
+)
+def test_server_timeout_before_content_error(
```

Comment:
This test doesn't have any asserts: I'd expect it to use something like `capfd` fixture and check what server outputs.